src/utils/structure.js

```diff
@@ -1,50 +1,102 @@
-const items = [
-  {
-    description: 'Doctor appointment at 10AM',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Visit friends home at 12PM',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Complete project for Microverse',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Take programming tutorials on udemy',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Dinner plan with family at hotel',
-    completed: false,
-    index: 4,
-  },
-];
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Kindly remove this eslint disable line its best to fix the issue in the code.